<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMSearchGroup</class>
 <widget class="QWidget" name="IMSearchGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMSearchGroup</string>
  </property>
  <widget class="CFriendListWidgetBase" name="mListWidgetSearchGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>700</width>
     <height>325</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mlineEditSearchGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>381</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mPButtonSearchGroup">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mLabelSearchGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>mLabelSearchGroup</zorder>
  <zorder>mListWidgetSearchGroup</zorder>
  <zorder>mlineEditSearchGroup</zorder>
  <zorder>mPButtonSearchGroup</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CFriendListWidgetBase</class>
   <extends>QListWidget</extends>
   <header>cfriendlistwidgetbase.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
